framework/logging.cpp

```diff
@@ -1948,6 +1949,7 @@ std::string TapFormatLogger::fail_info_details()
     case EX_PROTOCOL:   return "remote error in protocol";
     case EX_NOPERM:     return "permission denied";
     case EX_CONFIG:     return "configuration error";
+    case EXIT_MEMORY:   return "Failed to perform an action due to memory shortage.";
```

Comment:
Strings are capitalized inconsistently (new ones are, the existing are not).

Reply:
They come from different sources... Fixed.